Dockerfile

```diff
@@ -67,6 +67,9 @@ RUN playwright install --with-deps chromium
 
 RUN apt-get update && apt-get install -y --no-install-recommends libpq-dev wget grep
 
+# install claude code
+curl -fsSL https://claude.ai/install.sh | bash
+
```

Comment:
@Nihal-Srivastava05 this is not a valid docker command. When making changes to Dockerfiles, please check if the docker build works.